src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -511,18 +514,16 @@ private static IdTitleViewModel BuildPublicationViewModel(PublicationMethodology
             return new IdTitleViewModel(publication.Id, publication.Title);
         }
 
-        private async Task<Either<ActionResult, Unit>> ValidateMethodologySlugUniqueForUpdate(
-            MethodologyVersion methodologyVersion, string slug)
+        private async Task<Either<ActionResult, Unit>> ValidateMethodologySlugUnused(string slug)
```

Comment:
This name doesn't make a lot of sense. `ValidateMethodologySlugUnique` would be a more appropriate name.